css/styles.css

```diff
@@ -6,10 +6,14 @@ body {
     font-family: 'Press Start 2P', cursive;
 }
 
-.container {
+#container {
```

Comment:
Id should be for hooking on JS not CSS, use ```<main>``` as discussed

Reply:
🎉